controller/src/main/resources/reference.conf

```diff
@@ -33,6 +33,7 @@ CONTAINER_COUNT = 4
 REST_SERVER_IP = "0.0.0.0"
 REST_SERVER_PORT = 9091
 
+MIN_LEASE_VALUE = 5000
```

Comment:
The minimum interval at which Pinger sends pings() is 10 seconds. { ref: https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/stream/impl/Pinger.java#L79 }
This implies a min lease of 20 seconds, right?

Reply:
I'm fine either way, with increasing the minimum controller side or dropping the minimum client side, but clearly we should fix one of them.

Reply:
Are you saying one ping is sent every 10 seconds for a transaction?
Then we did not need a minimum lease because any lease below the ping interval will anyway expire unless client issues a commit before that. 
But we can set this to 10 seconds as the default value

>  This implies a min lease of 20 seconds, right?
I did not get this. Why would minimum lease need to be 20 seconds. It could be anything greater than 10 seconds so that client gets an opportunity to implicitly renew lease.

Reply:
> I did not get this. Why would minimum lease need to be 20 seconds. It could be anything greater than 10 seconds so that client gets an opportunity to implicitly renew lease.

Because the minimum rate is 1 every 10s, and the client issues pings at 0.5 of the lease, so 20s is the minimum allowed lease time according to the client today.

> I did not get this. Why would minimum lease need to be 20 seconds. It could be anything greater than 10 seconds so that client gets an opportunity to implicitly renew lease.

I'd rather have a minimum controller side to be safe. Even if the client behaves like this today, we never know what kind of bugs or changes we can introduce later that would break things.

Reply:
> Because the minimum rate is 1 every 10s, and the client issues pings at 0.5 of the lease, so 20s is the minimum allowed lease time according to the client today.
so does the client say lets take ping frequency = `max(0.5*lease, 10)`  
if so, then a value of anything greater than 10 is also fine. anything close to 10 becomes a close call like say with 11 we only get 1 second to make a call to controller and renew the lease.

anyway, a minimum lease time of 20 seconds is reasonable default. should i update the PR with it?

Reply:
No, let's change to:

1- Min. ping period = 5s
2- Min. lease time = 10s.
3- Max. lease time = 120s.

There is no reason to limit the lease time to be as short as 30s.

Reply:
done